gardenhealth/checker.go

```diff
@@ -129,7 +129,7 @@ func (c *checker) create(logger lager.Logger) (string, garden.Container, error)
 	logger.Debug("starting")
 	defer logger.Debug("finished")
 
-	guid := HealthcheckPrefix + c.guidGenerator.Guid(logger)
+	guid := c.guidGenerator.Guid(HealthcheckPrefix, logger)
```

Comment:
We like to have `logger` always be the first argument to functions. Can you swap those?